Engine/Src/Ancona/System/Android/FileOperations.cpp

```diff
@@ -82,12 +82,6 @@ std::istream * AndroidFileOperations::GetAndroidFileInputStream(const std::strin
     {
         auto apkFileStream = OpenFile(desiredFile);
         Assert(apkFileStream != nullptr, "Could not find the " + desiredFile + " file in app storage or within apk.");
-
```

Comment:
We're still saving map files after serialization in [MapSerializer.cpp:155](https://github.com/ild-games/Ancona/blob/ae8583d3919cfb757c2b8b29608994d12ef3b748/Engine/Src/Ancona/Framework/Serializing/MapSerializer.cpp#L155), should we remove this since we don't use this feature ever? It will either Assert here on android or save the copy to external space anyway. We should probably stop doing it.

Reply:
I don't think this is ever called with _loading = true. I think cleaning up all the dead saving code should be done in its own PR.